lib/galaxy/model/__init__.py

```diff
@@ -1157,7 +1177,7 @@ def is_hda(d):
     return isinstance( d, HistoryDatasetAssociation )
 
 
-class History( object, Dictifiable, UsesAnnotations, HasName ):
+class History( HasTags, Dictifiable, UsesAnnotations, HasName ):
```

Comment:
It works the same way, but I prefer the more explicit `object`, if we could leave that here.  HasTags must come before Dictifiable right?  MRO mess is why we may have pulled back from some of these mixins, IIRC.